File: Lab3OOP/Composition.cpp
#include "Composition.h"
#include "iostream"
#include <sstream>

Composition::Composition(const std::string& title, int duration) : title(title), duration(duration) {}

std::string Composition::getTitle() const {
    return title;
}

int Composition::getDuration() const {
    return duration;
}

std::string  Composition::show() const {
    std::ostringstream oss;
    oss << "Title: " << title << ", Duration: " << duration << " sec" << std::endl;
    return oss.str();
}

File: Lab3_oop_python/composition.py
class Composition:
    def __init__(self, title: str, duration: int):
        self.title = title
        self.duration = duration

    def get_title(self) -> str:
        return self.title

    def get_duration(self) -> int:
        return self.duration
    def show(self):
        return((f"Title: {self.title}, Duration: {self.duration} sec"))

